Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Webmaker API
+
+FROM ubuntu:14.04
```

Comment:
I think it's better if we set this to `14.04.2`, since there will be less updates to install compared to 14.04